applications/serial_lte_modem/src/gpio/Kconfig

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+config SLM_GPIO
+	bool "GPIO support in SLM"
+	default n
+	select GPIO
```

Comment:
Default should be 'y' as required by PMT
CI team does not want to see "select", change to "depends on" and select in prj.conf
BTW, CONFIG_GPIO has already been configured in prj.conf.

Reply:
Modify default to y.
Change to depends on GPIO